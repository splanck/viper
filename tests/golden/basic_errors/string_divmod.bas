10 PRINT "A" \ 1
20 PRINT "B" MOD 1
30 END
