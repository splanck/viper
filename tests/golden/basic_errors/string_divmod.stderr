B2001.*B2001
